modules/kinesis/src/main/scala/com/snowplowanalytics/snowplow/enrich/kinesis/Sink.scala

```diff
@@ -238,23 +280,19 @@ object Sink {
     kinesis.putRecords(putRecordsRequest)
   }
 
-  private def getErrorsSummary(badResponses: List[PutRecordsResultEntry]): Map[String, (Long, String)] =
-    badResponses.foldLeft(Map[String, (Long, String)]())((counts, r) =>
-      if (counts.contains(r.getErrorCode))
-        counts + (r.getErrorCode -> (counts(r.getErrorCode)._1 + 1 -> r.getErrorMessage))
-      else
-        counts + (r.getErrorCode -> ((1, r.getErrorMessage)))
-    )
+  private def failureMessageForInternalErrors(
+    records: List[PutRecordsRequestEntry],
+    streamName: String,
+    result: TryBatchResult
+  ): String = {
+    val exampleMessage = result.exampleInternalError.getOrElse("none")
+    s"Writing ${records.size} records to $streamName errored with internal failures. Example error message [$exampleMessage]"
+  }
 
-  private def logErrorsSummary[F[_]: Sync](errorsSummary: Map[String, (Long, String)]): F[Unit] =
-    errorsSummary
-      .map {
-        case (errorCode, (count, sampleMessage)) =>
-          Logger[F].error(
-            s"$count records failed with error code ${errorCode}. Example error message: ${sampleMessage}"
-          )
-      }
-      .toList
-      .sequence_
+  private def failureMessageForThrottling(
+    records: List[PutRecordsRequestEntry],
+    streamName: String
+  ): String =
+    s"Exceeded Kinesis provisioned throughput: ${records.size} records failed writing to $streamName."
```

Comment:
I think that this is fine to not add the error messages to know about the shard IDs because I think that we can see throughput exceeded metric per shard in CloudWatch